.gitignore

```diff
@@ -1,2 +1,21 @@
-node_modules
-.vscode/*
+## IDE specific
+.idea/
+.DS_Store
+DS_Store?
+.vscode/
+
+## Building
+node_modules/
+dist/
+build/
+.cache
+
+# Logs
+logs
+*.log*
+
+## Other
+Desktop.ini
+Thumbs.db
+._*
+tmp/
```

Comment:
empy line again. and everywhere I think. Github yelling on us :)

Reply:
done